inc/session.class.php

```diff
@@ -575,11 +575,12 @@ static function loadLanguage($forcelang='') {
       if (isset($CFG_GLPI["languages"][$trytoload][5])) {
          $_SESSION['glpipluralnumber'] = $CFG_GLPI["languages"][$trytoload][5];
       }
-      $TRANSLATE = new Zend\I18n\Translator\Translator;
```

Comment:
I do not see the point removing this here and adding two occurrences in the try and the catch below.